### PR TITLE
[FIX JENKINS-50748] Handle null arg to jsStringEscape

### PR DESCRIPTION
**Pull request by daniel-beck**
See [JENKINS-50748](https://issues.jenkins-ci.org/browse/JENKINS-50748).

### Proposed changelog entries

* Bug: Don't log null pointer exceptions on some forms with validation button (regression in 2.116).

@Wadeck @oleg-nenashev @dwnusbaum

### CODE REVIEW

File: core/src/main/java/hudson/Functions.java
Comment (oleg-nenashev) [inline core/src/main/java/hudson/Functions.java]: Makes sense to add `Nullable` annotations and Javadoc

**Review by oleg-nenashev — APPROVED** [review]

**Comment by daniel-beck** [comment]
An alternative (or possible addition) to this would be to revert this part of the fix. It's not clear that this is actually needed for the security fix; as the form fields for validation are not typically user controlled.

**Review by olivergondza — APPROVED** [review]

**Review by Wadeck — APPROVED** [review]
:bee: LGTM, tested before and after fix, problem seems solved.

> as the form fields for validation are not typically user controlled

My only concern about that is that we provide some tools and we do not know what are the usages maintainers could have and so providing your fix seems a better approach from my PoV.

**Review by dwnusbaum — APPROVED** [review]

**Pull request merged**
